Dockerfile

```diff
@@ -12,6 +12,7 @@ RUN set -ex; \
         curl \
         lxc \
         iptables \
+        liblzma-dev \
```

Comment:
Oh, this is a leftover from when context.py was in compose, there's no actual need for this dependency here. The lzma compression is handled as a special case in docker-py now: under py3 the tarfile module has built-in support for lzma (presuming the xz-utils headers where available when interpreter was built) but under py2 docker-py just accepts a file with an '.xz' extension as a lzma compressed tarball and passes it on to the build call without 'peeking' into it.